src/analysis/complex/basic.lean

```diff
@@ -116,6 +116,13 @@ by rw [dist_comm, dist_conj_self]
 lemma nndist_self_conj (z : ℂ) : nndist z (conj z) = 2 * real.nnabs z.im :=
 by rw [nndist_comm, nndist_conj_self]
 
+protected lemma dist_inv_inv (z w : ℂ) (hz : z ≠ 0) (hw : w ≠ 0) :
```

Comment:
This statement is true for any normed_field, right?

Reply:
Good point, yes. The following should probably go about [here](https://github.com/leanprover-community/mathlib/blob/ef7acf407d265ad4081c8998687e994fa80ba70c/src/analysis/normed/field/basic.lean#L578): 
```
import analysis.normed.field.basic

namespace normed_field

variables {α : Type*} [normed_field α]

protected lemma dist_inv_inv (z w : α) (hz : z ≠ 0) (hw : w ≠ 0) :
  dist z⁻¹ w⁻¹ = (dist z w) / (‖z‖ * ‖w‖) :=
by rw [dist_eq_norm, inv_sub_inv hz hw, norm_div, dist_eq_norm', norm_mul]

end normed_field
```

Reply:
A `normed_division_ring` version would be nice too (using the primed `inv_sub_inv'`)

Reply:
In fact I assume the statement would end  up the same, since the division is in `real`; so we only need one lemma, but proved in the more general case.

Reply:
we should have the corresponding `nndist` lemma too.